.gitignore

```diff
@@ -122,7 +122,7 @@ sorc/wafs_gcip.fd
 sorc/wafs_grib2_0p25.fd
 sorc/wafs_makewafs.fd
 sorc/wafs_setmissing.fd
-
+sorc/upp.fd
```

Comment:
how is this related to this PR?

Reply:
No relate to this PR, so I will delete it.